### PR TITLE
Dropdown component

### LINKED ISSUE #46: Arithmetic calculations input dropdown
screen shot 2016-02-04 at 15 10 41

### PR DESCRIPTION
**Pull request by MariaCheca**
Closes #46 

New dropdown component in forms family:
screen shot 2016-02-09 at 6 40 18 pm

CR @piensaenpixel

### CODE REVIEW

File: src/scss/cdb-components/forms/dropdowns.scss [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+.CDB-DropdownContainer {
+  position: relative;
+  height: 250px;
+}
+.CDB-Dropdown {
+  @include display-flex();
+  @include flex-direction(row);
+  position: absolute;
+  top: 40px;
+  height: 200px;
```
Comment (piensaenpixel) [inline src/scss/cdb-components/forms/dropdowns.scss]: IMO this height is no necessary, we should add a max-height to 'Dropdown-options'

File: src/scss/cdb-components/forms/dropdowns.scss [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  border-radius: 3px 0 0 3px;
+  background-color: $cSecondaryBackground;
+  box-sizing: border-box;
+}
+.CDB-Dropdown-calculationsElement {
+  font-size: $sFontSize-small;
+
+  &:not(:last-child) {
+    margin-bottom: $sLineHeight-medium;
+  }
```
Comment (piensaenpixel) [inline src/scss/cdb-components/forms/dropdowns.scss:79]: make easier

File: src/scss/cdb-components/forms/dropdowns.scss [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  top: 40px;
+  height: 200px;
+}
+.CDB-Dropdown-calculations {
+  display: inline-block;
+  width: $baseSize * 11;
+  height: 100%;
+  padding: $sMargin-element;
+  border-right: 1px solid $cMainLine;
+  border-radius: 3px 0 0 3px;
```
Comment (piensaenpixel) [inline src/scss/cdb-components/forms/dropdowns.scss]: Always 4px for border-radius

File: src/scss/cdb-components/forms/dropdowns.scss [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+  font-size: $sFontSize-small;
+
+  &:not(:last-child) {
+    margin-bottom: $sLineHeight-medium;
+  }
+}
+.CDB-Dropdown-options {
+  display: inline-block;
+  width: $baseSize * 20;
+  border-radius: 0 3px 3px 0;
```
Comment (piensaenpixel) [inline src/scss/cdb-components/forms/dropdowns.scss]: Same here

File: src/scss/cdb-components/forms/dropdowns.scss [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+    border-bottom: 0;
+  }
+  &:hover {
+    background-color: rgba(157, 224, 173, 0.2);
+    cursor: pointer;
+  }
+}
+.CDB-Dropdown-optionsElement.is-selected {
+  color: $cBlack;
+}
```
Comment (piensaenpixel) [inline src/scss/cdb-components/forms/dropdowns.scss:104]: are you sure is black? or mainDark?

File: src/scss/cdb-components/forms/dropdowns.scss [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+.CDB-Dropdown {
+  @include display-flex();
+  @include flex-direction(row);
+  position: absolute;
+  top: 40px;
+  height: 200px;
+}
+.CDB-Dropdown-calculations {
+  display: inline-block;
```
Comment (piensaenpixel) [inline src/scss/cdb-components/forms/dropdowns.scss]: not necessary

File: src/scss/cdb-components/forms/dropdowns.scss [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+  width: $baseSize * 11;
+  height: 100%;
```
Comment (piensaenpixel) [inline src/scss/cdb-components/forms/dropdowns.scss]: not necessary

File: src/scss/cdb-components/forms/dropdowns.scss [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+.CDB-Dropdown {
+  @include display-flex();
+  @include flex-direction(row);
+  position: absolute;
+  top: 40px;
+  height: 200px;
+}
+.CDB-Dropdown-calculations {
+  display: inline-block;
+  width: $baseSize * 11;
```
Comment (piensaenpixel) [inline src/scss/cdb-components/forms/dropdowns.scss]: I'm not very sure about this, i think we should remove it

File: src/scss/cdb-components/forms/dropdowns.scss [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+// ----------------------------------------------
+
+/* SG
+# Forms/Dropdowns
+
+```
+<div class="CDB-DropdownContainer">
+  <input type="text" name="text" placeholder="DejaVu Sans" class="CDB-InputText CDB-Text">
+  <div class="CDB-Dropdown CDB-Box-Modal">
+    <div class="CDB-Dropdown-calculations CDB-Text is-semibold">
```
Comment (piensaenpixel) [inline src/scss/cdb-components/forms/dropdowns.scss]: Try add this classes to the ul and remove the <div>

File: src/scss/cdb-components/forms/dropdowns.scss [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+        </li>
+        <li class="CDB-Dropdown-calculationsElement u-iBlock">
+          <input class="CDB-Radio" type="radio" name="calculation" value="05">
+          <span class="u-iBlock CDB-Radio-face"></span>
+          <label class="u-iBlock u-lSpace">MIN</label>
+        </li>
+      </ul>
+    </div>
+
+    <div class="CDB-Dropdown-options CDB-Text">
```
Comment (piensaenpixel) [inline src/scss/cdb-components/forms/dropdowns.scss]: Same here

File: src/scss/cdb-components/forms/dropdowns.scss [hunk 11]
```diff
@@ -0,0 +1,119 @@
...
+/* SG
+# Forms/Dropdowns
+
+```
+<div class="CDB-DropdownContainer">
+  <input type="text" name="text" placeholder="DejaVu Sans" class="CDB-InputText CDB-Text">
+  <div class="CDB-Dropdown CDB-Box-Modal">
+    <div class="CDB-Dropdown-calculations CDB-Text is-semibold">
+      <ul>
+        <li class="CDB-Dropdown-calculationsElement u-iBlock">
```
Comment (piensaenpixel) [inline src/scss/cdb-components/forms/dropdowns.scss]: u-iBlock is no necessary in all the items

File: src/scss/cdb-components/forms/dropdowns.scss [hunk 12]
```diff
@@ -0,0 +1,119 @@
...
+
+    <div class="CDB-Dropdown-options CDB-Text">
+      <ul>
+        <li class="CDB-Dropdown-optionsElement is-selected">
+          <span class="u-iBlock u-lSpace CDB-Size-medium">selected</span>
+        </li>
+        <li class="CDB-Dropdown-optionsElement">
+          <span class="u-iBlock u-lSpace CDB-Size-medium">active</span>
+        </li>
+        <li class="CDB-Dropdown-optionsElement is-disabled">
```
Comment (piensaenpixel) [inline src/scss/cdb-components/forms/dropdowns.scss]: if the item is disabled you can't do hover

File: src/scss/cdb-components/forms/dropdowns.scss [hunk 13]
```diff
@@ -0,0 +1,116 @@
...
+    </div>
+  </div>
+</div>
+```
+*/
+
+@import '../../cdb-utilities/mixins';
+@import '../../cdb-variables/sizes';
+@import '../../cdb-variables/colors';
+@import '../../cdb-utilities/mixins';
```
Comment (xavijam) [inline src/scss/cdb-components/forms/dropdowns.scss]: Watch out!

**Comment by MariaCheca** [comment]
:sparkles:

**Pull request merged**